controllers/operatorconfigmap_controller.go

```diff
@@ -194,80 +229,55 @@ func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			"config.json": "[]",
 		},
 	}
-	if err := c.own(monConfigMap); err != nil {
+	if err := c.own(encConfigMap, operatorConfigMap); err != nil {
 		return ctrl.Result{}, err
 	}
-	err = c.create(encConfigMap)
-	if err != nil && !k8serrors.IsAlreadyExists(err) {
+	if err := c.create(encConfigMap); err != nil && !k8serrors.IsAlreadyExists(err) {
 		c.log.Error(err, "failed to create monitor configmap", "name", encConfigMap.Name)
 		return ctrl.Result{}, err
 	}
 
-	c.cephFSDeployment = &appsv1.Deployment{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      csi.CephFSDeploymentName,
-			Namespace: c.OperatorNamespace,
-		},
-	}
 	err = c.createOrUpdate(c.cephFSDeployment, func() error {
-		if err := c.own(c.cephFSDeployment); err != nil {
+		c.cephFSDeployment = csi.GetCephFSDeployment(c.OperatorNamespace)
+		if err := c.own(c.cephFSDeployment, operatorConfigMap); err != nil {
 			return err
 		}
-		csi.GetCephFSDeployment(c.OperatorNamespace).DeepCopyInto(c.cephFSDeployment)
 		return nil
```

Comment:
not sure why golint didn't suggest this, just `return c.own(c.cephFSDeployment, operatorConfigMap)` same for remaining.